src/app/map/_components/map-menu/MapMenuExtraRoomFilterItem.tsx

```diff
@@ -0,0 +1,50 @@
+import { Dispatch, SetStateAction } from "react";
+
+type MapMenuExtraRoomFilterItemProps = {
+  type: string;
+  activeRooms: string[];
+  setActiveRooms: Dispatch<SetStateAction<string[]>>;
+};
+
+function setColor(type: string, activeRooms: string[]) {
+  if (activeRooms.includes(type)) {
```

Comment:
Could we do an early return here by inverting the condition?
'''
If (!activeRooms.includes(type)) return "white" 
'''